### PR TITLE
merge date 10.06.2015: Removed ruby1.9 support

### PR DESCRIPTION
**Pull request by goldhoorn**
Remove compatibility to 1.9

### CODE REVIEW

**Comment by goldhoorn** [comment]
This should not merged before 10.05.2015

**Comment by goldhoorn** [comment]
Btw. 
why do we have this bootstrap.sh scripts at all i dont like them. 
They are copied around, and from a maintainer sight we have no way to adapt them.
The branch settings get more complex. Why we not simply tell the customers howto bootstrap with using the autoproj_bootstrap script and removing this file at all?

**Comment by goldhoorn** [comment]
wah wrong PR (again) sorry

File: init.rb [hunk 1]
```diff
@@ -12,15 +12,6 @@
 # Ignore the numerous backup files
 ignore(/~$/)
 
-# Verify that Ruby is NOT 1.8. 1.8 is unsupported
-if defined?(RUBY_VERSION) && (RUBY_VERSION =~ /^1\.8\./)
```
Comment (doudou) [inline init.rb]: Should keep this for stable, no ?

**Comment by goldhoorn** [comment]
*updated*

I will apply this patch on 10.06.2015 as discussed

File: init.rb [hunk 2]
```diff
@@ -12,12 +12,13 @@
 # Ignore the numerous backup files
 ignore(/~$/)
 
-# Verify that Ruby is NOT 1.8. 1.8 is unsupported
+
+# Ruby 1.8 is completly outdated, if you modify this, take respect to the addition checks below against 1.9 
 if defined?(RUBY_VERSION) && (RUBY_VERSION =~ /^1\.8\./)
```
Comment (2maz) [inline init.rb]: Looks like this section can be removed, since the next is checking on Ruby < 2.0

Comment (goldhoorn) [inline init.rb]: Is is still needed because stable can be still 1.9

**Pull request merged**